File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //assigns h to the height inputted by the user
    int h;

    //continues to prompt user until a positive number between 1 and 8 is entered
    do
    { 
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //double loop to build blocks, first loop builds the rows, second loop builds the columns
    for (int i = h; i > 0; i--)
    {
        for (int j = (i - 1); j > 0; j--)
        {
            printf(" ");
        }
        for (int k = (h - (i - 1)); k > 0; k--)
        {
            printf("#");  
        }
        printf("\n");
    }
}

File: pset6/mario-less/mario.py
from cs50 import get_int

h = 0

while (h < 1 or h > 8):
    h = get_int("Height: ")

for i in range(int(h)):
    h -= 1
    i += 1
    print(" " * h, end="")
    print("#" * i, end="")
    print("")

"""
while h > 0:
    h -= 1
    i += 1
    print(" " * h, end="")
    print("#" * i, end="")
    print("")
"""